src/sections/Blog/Blog-sidebar/blogSidebar.style.js

```diff
@@ -222,6 +222,9 @@ const BlogSideBarWrapper = styled.div`
                 .logo{
                  width: 100%;  
                  }
+                .logo-top-margin{
+                    margin-top: 10px
```

Comment:
can we use rem here?

Reply:
@abhijeetgauravm yes, should I change it rem and push the changes?

Reply:
yeah @toth2000 that will be good!

Reply:
@abhijeetgauravm done